Console/retro/Console.h

```diff
@@ -27,6 +27,9 @@
 #include <vector>
 #include <string>
 
+#define BEL 7
```

Comment:
please use `const char BEL = 7;` and move to the source file. No one else needs to know.
Gobal defines are out of fashion ;-)

Reply:
Indeed! I should have done that from the start.